### DISCUSSION TITLE
OctoDash and Octoprint/Web at the same time

### DISCUSSION THREAD

**Discussion by ralfsteck** [UnchartedBull/OctoDash#3995] [Q&A] [answered]
Maybe I misunderstand it completely, but when I install OctoDash on the Pi to work directly at the printer, can I still connect to the stock Octoprint web interface with other computers (e.g. my notebook in the bureau) or do those also show Octodash?

**Comment by jcleek** [comment, marked as answer]
Yes, you can still see and use OctoPrint on other computers
